js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts

```diff
@@ -121,6 +123,18 @@ export const AudioAttributesPostfixes = {
   transcript: "transcript",
 } as const;
 
+export const AgentPostfixes = {
+  name: "name",
+  step: "step",
+  next_agent: "next_agent",
+} as const;
+
+export const GraphPostfixes = {
+  node_name: "node.name",
+  node_step: "node.step",
+  node_next_node: "node.next_node",
+} as const;
```

Comment:
this might be me, but just adding "graph" semcov seems like the right idea. It might be difficult to try to mix the two.
```suggestion
export const GraphPostfixes = {
  node_name: "node.name",
  node_step: "node.step",
  node_next: "node.next",
} as const;
```

Reply:
Not sure I understand the difference between the graph and agent semconv. I assume the graph is intended to capture the set of nodes and edges, regardless of whether those edges were actually traversed in a particular trajectory. Are agent conventions meant to capture actual trajectories?